src/xz/chevahbs

```diff
@@ -37,7 +37,10 @@ chevahbs_try() {
 
 
 chevahbs_cp() {
-    execute "${MAKE_CMD[@]}" install DESTDIR="$INSTALL_DIR"
+    # xz's installation copies binaries too, but there's no need for them.
+    execute cp -R src/liblzma/api/* "$INSTALL_DIR"/include/
```

Comment:
Do we need these files in the install dir ?

What is the "install dir" ?

I expect them to be needed in the build dir , but they should not be needed for the final package

Reply:
`$INSTALL_DIR` is where the built libraries and Python are installed (or simply copied as needed).

Pruning files needed there but not wanted in the final dist package is done through `cleanup_install_dir()` in `functions_build.sh`: https://github.com/chevah/pythia/blob/ba80a304e8d5ce7c02d02fe419642aefed5da907/functions_build.sh#L172

Reply:
Added https://github.com/chevah/pythia/issues/67 for this.